canonical-web-and-design/assets.ubuntu.com
# Create basic flask app structure

## Done
Create initial structure to start porting the application to flask.

## QA
- ./run
- go to http://0.0.0.0:8017 and see a `401`
- go to http://0.0.0.0:8017/?token=correcthorsebatterystaple and see a `200 {}`
